src/addressBook/index.js

```diff
@@ -337,13 +305,8 @@ export function createContact(addressBookIri, contact, types) {
   };
 }
 
-export async function findContactInAddressBook(addressBookIri, webId, fetch) {
-  const { response: people } = await getContacts(
-    foaf.Person,
-    addressBookIri,
-    fetch
-  );
-
+export async function findContactInAddressBook(peopleDataset, webId, fetch) {
```

Comment:
Same here and everywhere else in this file - ideally this would be generic rather than use the word `people`, as it would also work for organizations, apps, etc.